src/main/resources/proton.accessWidener

```diff
@@ -1,4 +1,4 @@
 accessWidener   v1  named
 
 # Methods
-accessible  method net/minecraft/block/Blocks   never   (Lnet/minecraft/block/BlockState;Lnet/minecraft/world/BlockView;Lnet/minecraft/util/math/BlockPos;)Z
\ No newline at end of file
+accessible  method  net/minecraft/block/Blocks  never   (Lnet/minecraft/block/BlockState;Lnet/minecraft/world/BlockView;Lnet/minecraft/util/math/BlockPos;)Z
```

Comment:
tabs are better imo

Reply:
I did tabs, I think IntelliJ made them into spaces

Reply:
oh concern. we need checkstle to fix that. probably out of the scope of this pr though.